docs/template_plugin/tests/functional/op_reference/non_max_suppression.cpp

```diff
@@ -1528,9 +1534,6 @@ std::vector<NonMaxSuppression1Params> generateParams1() {
 
 std::vector<NonMaxSuppression1Params> generateCombinedParams1() {
     const std::vector<std::vector<NonMaxSuppression1Params>> generatedParams {
-        generateParams1<element::Type_t::bf16, element::Type_t::i32, element::Type_t::f32, element::Type_t::i32>(),
```

Comment:
Why did you remove these test cases?

Reply:
Because NonMaxSuppression-3,4,5 has its type T_IND which is i32 or i64, but NonMaxSuppression1 does not specify the type and specify integer in its documentation, and the default type seems to be i64, so i32 tests in NonMaxSuppression-1 are removed instead skipping them in skip_tests_config.cpp.